api_tests/tests/erc20_halight.ts

```diff
@@ -16,15 +16,15 @@ it(
         await alice.createHerc20HalightSwap(bodies.alice);
         await bob.createHerc20HalightSwap(bodies.bob);
 
-        await alice.init();
+        await expect(alice.next()).resolves.toBe("init");
```

Comment:
Could we reduce the noise in writing tests here by introducing something like `alice.assertAndExecuteNext("init")`.

Naming of the function subject to be improved :)

Reply:
Yes, just wanted to check first that my peers agree on the assertion of the step.

Reply:
Oh definitely, I actually wanted to make the above a question but forgot the question mark, lol :D